### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by vsmutok**
Introduced multiple test cases using pytest to verify the functionality of the `copy_file` function. These include handling successful file copying, invalid commands, missing files, and edge cases such as identical source and target names or empty commands.

### CODE REVIEW

**Review by bvsn — APPROVED** [review]

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -26,3 +26,6 @@ jobs:
       - name: Run flake8
         run: flake8 app/
 
+      - name: Run tests
+        timeout-minutes: 5
+        run: pytest tests/
```
Comment (bvsn) [inline .github/workflows/test.yml:31]: Can't get `app.main`, because it isn't a package.

File: tests/test_main.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+    copy_file("cp only_one_argument.txt")
+    assert not os.path.exists(target_filename)
+
+
+def test_empty_command(setup_and_teardown_files):
+    """Test that the function does nothing for an empty command."""
+    _, target_filename, _ = setup_and_teardown_files
+
+    copy_file("")
```
Comment (bvsn) [inline tests/test_main.py:75]: IndexError: list index out of range

**Review by bvsn — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (vsmutok) [reply .github/workflows/test.yml:31]: @bvsn  це очікувана логіка, так як в модулі `app.main` просто немає функції `copy_file`
якщо я щось не вірно розумію дай знати

File: tests/test_main.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
+import os
+
+import pytest
+
+from ...app.main import copy_file
```
Comment (vsmutok) [inline tests/test_main.py:5]: @bvsn як на мене `..` зайві тут

File: tests/test_main.py [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+    copy_file("invalid_command test_source.txt test_target.txt")
+    assert not os.path.exists(target_filename)
+
+
+def test_missing_source_file(setup_and_teardown_files):
+    """Test that the function handles missing source file gracefully."""
+    _, target_filename, _ = setup_and_teardown_files
+
+    copy_file("cp non_existing_file.txt test_target.txt")
```
[the remark this answers is not in the document]
Comment (vsmutok) [reply tests/test_main.py:59]: @bvsn тут пліз поясни, в описі завдання ніде не сказано що код повинен викликати певні помилки, тож я писав тести відштовхуючись від умови завдання

File: .github/workflows/test.yml (hunk 1 above)
Comment (bvsn) [reply .github/workflows/test.yml:31]: Тести не запускаються, тому думав виправити це через відносний імпорт, але не допомогло.

File: tests/test_main.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (bvsn) [reply tests/test_main.py:59]: Тест падає з такою помилкою тому pipeline не завершується успішно.

File: .github/workflows/test.yml (hunk 1 above)
Comment (vsmutok) [reply .github/workflows/test.yml:31]: @bvsn я додав демо рішення, все відпрацювало так як очікувано, не знаю чому воно в тебе сипалось...

**Review by bvsn — APPROVED** [review]
